contrib/seeds/generate-seeds.py

```diff
@@ -100,7 +100,10 @@ def parse_spec(s):
 
     host = name_to_bip155(host)
 
-    return host + (port, )
+    if host[0] == BIP155Network.TORV2:
+        return -1  # TORV2 is no longer supported, so we ignore it
```

Comment:
Would it be possibly better to return `None` here?

Reply:
Yes, I asked this in https://github.com/bitcoin/bitcoin/pull/22050#issuecomment-850808537, happy to update if reviewers think it's worth changing.

Reply:
Ah, sorry. `None` +1 from me :)

Reply:
Ok, I have a couple minor refactorings ready that aren't essential to removing tor v2 support and propose to insert this change into that commit, unless I need to update this patch for another reason.

Reply:
done